internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -535,6 +554,42 @@ if ($with_fluentd) {
     echo "- Started"
 }
 
+if ($with_dotnet_instrumentation) {
+    echo "Installing SignalFx Instrumentation for .NET ..."
+    $api = "https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest"
+    $module_name = "Splunk.SignalFx.DotNet.psm1"
+    echo "Downloading .NET Instrumentation installer ..."
+    $download = (Invoke-WebRequest $api | ConvertFrom-Json).assets | Where-Object { $_.name -like $module_name } | Select-Object -Property browser_download_url,name
+    $dotnet_auto_path = Join-Path $tempdir $download.name
+    Invoke-WebRequest -Uri $download.browser_download_url -OutFile $dotnet_auto_path
+    Import-Module $dotnet_auto_path
+    echo "Installing SignalFx Dotnet Auto Instrumentation..."
+    Install-SignalFxDotnet
+
+    echo "Setting environment variables for instrumentation ..."
+    update_registry -path "$regkey" -name "COR_ENABLE_PROFILING" -value "1"
+    update_registry -path "$regkey" -name "COR_PROFILER" -value "{B4C89B0F-9908-4F73-9F59-0D77C5A06874}"
+    update_registry -path "$regkey" -name "CORECLR_ENABLE_PROFILING" -value "1"
+    update_registry -path "$regkey" -name "CORECLR_PROFILER" -value "{B4C89B0F-9908-4F73-9F59-0D77C5A06874}"
```

Comment:
@pjanotti @pellared @pmcollins since we already have PS module. I'm thinking of moving these 4 variables also to the newly created module. Something like `Register-SignalFxForGlobal`. This bit seems to be the last specifics to the dotnet, so collector's script will be very clean and without any specific's knowledge. 

+ There is a chance that cor profiler's guid will change. Eg when implementing multiplexer etc. So, any possibility to change should be in version specific setup script.

Reply:
But what are we trying to achieve here? Instrument `csrss.exe`?

Reply:
No, I haven't tested this extensively and was hoping to get more info from your team about performance implications and a testing strategy. I hope you can make it to tomorrow's meeting about this at 12:30 eastern so we can discuss.